assets/javascript/petapp.js

```diff
@@ -301,14 +296,16 @@ $(document).ready(function() {
         var phone = response.listings[i].phone.dispNum;
         var geoCode = response.listings[i].geoCode.latitude
           + response.listings[i].geoCode.longitude;
+        $('#displayGif').prepend(selectionDiv);
         var result = $("<p>")
           .html(name + "<br>" + "Address: " + address + "<br>" + "Phone: " + phone + "<br>" + "Website: " + resultUrl)
-          .appendTo($("#displayAPI"));
+          .appendTo($("#displayGif"));
```

Comment:
Hmm... where did your giphy API function go? It seems like you've merged the Yellow API and giphy API functions, but maybe github is only showing me part of the codebase in this review.